packages/frontend/src/components/Signin.vue

```diff
@@ -198,13 +205,19 @@ export default {
   },
   methods: {
     btnStatus() {
-      if (this.ssoOrCredentials == "Credentials" && this.username != "" && this.password != "")
-        this.disableButton = false;
-      else if (this.ssoOrCredentials == "SSO" && this.passcode != "") this.disableButton = false;
-      else this.disableButton = true;
+      if (this.endpoint && this.isCFEndpointValid) {
+        if (this.ssoOrCredentials == "Credentials" && this.username != "" && this.password != "") {
+          this.disableButton = false;
+        } else if (this.ssoOrCredentials == "SSO" && this.passcode != "") {
+          this.disableButton = false;
+        }
```

Comment:
I think you missed the scenario that you have a valid endpoint but one of the fields username/password/passcode is empty so we want the button to be disable

Reply:
fixed, thanks!